next.config.js

```diff
@@ -1,5 +1,6 @@
 /** @type {import('next').NextConfig} */
-const { i18n } = require('./next-i18next.config.js');
+import pkg from './next-i18next.config.js';
+const { i18n } = pkg;
 
```

Comment:
Please delete this configuration, we have already used other methods. Note that the front-end is the ES syntax.